.buildkite/pipeline.yml

```diff
@@ -1,9 +1,27 @@
 steps:
+  - label: 'stack rebuild'
+    env:
```

Comment:
This `env` section is from when we used `cache-s3` so maybe try removing it and seeing if things still work

Reply:
Done! Let's see...